src/signature.ts

```diff
@@ -8,7 +8,7 @@ const SIGNATURE_LENGTH = 64;
 export class Signature {
     private valueHex: string = "";
 
-    constructor(value?: string | Buffer) {
+    constructor(value?: string | Buffer | Uint8Array) {
```

Comment:
:+1:

Reply:
Should be handled below, similar to:

https://github.com/multiversx/mx-sdk-js-core/blob/main/src/address.ts

That is, use `ArrayBuffer.isView(value)`.

Reply:
done